src/couchapps/WMStats/_attachments/js/ViewModels/WMStats.ViewModel.js

```diff
@@ -433,6 +433,36 @@ WMStats.ViewModel = (function (){
             summary.SoftwareVersions = requestInfo.SoftwareVersions;
             summary.CMSSWVersion = requestInfo.SoftwareVersions[0];
             summary.Campaign = requestInfo.Campaign;
+            // assign more value for acdc assignment
+            summary.Team = requestInfo.Teams[0];
+            //TODO get the site white list from acdc db. (site might be overflowed)
+            //also change templete to handle multiple selection
+            //summary.SiteWhitelist = requestInfo.SiteWhitelist.join();
+            //summary.SiteBlacklist = requestInfo.SiteBlacklist.join();
+            summary.AcquisitionEra = requestInfo.AcquisitionEra;
+            summary.ProcessingString = requestInfo.ProcessingString;
+            summary.ProcessingVersion = requestInfo.ProcessingVersion;
+            summary.Dashboard = requestInfo.Dashboard;
+            summary.MergedLFNBase = requestInfo.MergedLFNBase;
+            summary.TrustSitelists = false;
```

Comment:
This only affects default value. At assign ops have review and change if they needed. I guess I can check whether it is for the top level task here and set the default value accordingly.
```
if toplevel_task:
   summary.TrustSitelists = requestInfo.TrustSitelists;
else:
   summary.TrustSitelists = false;
```
I will create the new issue.